README.md

```diff
@@ -77,6 +78,8 @@ To use this library in your project manually you may:
 
 ## Migration Guide
 
+Version 3.0.0 - Passing in an array of videos support removed. You should merge videos in advance instead. One way to merge videos is with [ffmpeg](https://www.ffmpeg.org/).
```

Comment:
Should we offer some more detail rather than just pointing to ffmpeg? I know I struggled with this for a while before finally getting it to work. I can take lead on this and create a short blog post or gist or something, and then we can link to it here (if you think that'd be helpful).

Reply:
Sure that would be great. Yea I struggled with it as well...and since I still don’t understand how my solution worked I deferred from going into detail haha

Reply:
created issue ✅

Reply:
Great, thanks! I'm kind of sick, and also busy with exams till Weds. We can probably merge this now, and I'll just tack on a link to a blog post when I have a chance (probably Weds).

Reply:
Yea just do this whenever